### PR TITLE
Add Molecules/Cards/CardProject to design sytem

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: components/atoms/tag/tag.utils.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ElementType } from "react";
+
+import { TagAvatarPort, TagIconPort, TagPort } from "components/atoms/tag/tag.types";
+
+export function isTagAvatar<C extends ElementType>(
+  tag: TagAvatarPort<C> | TagIconPort<C> | TagPort<C>
```
Comment (haydencleary) [inline components/atoms/tag/tag.utils.ts:6]: Can't use just use `TagPort<C>` as it is a union of `TagAvatarPort` and `TagIconPort` ?

File: components/atoms/typo/adapters/default/default.variants.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export const TypoDefaultVariants = tv({
   },
   variants: {
     size: {
-      xxs: { base: "text-xxs" },
+      xxs: { base: "text-[11px]" },
```
Comment (haydencleary) [inline components/atoms/typo/adapters/default/default.variants.ts:9]: If this is something that is in our design system maybe we should add a font size xxs to our Tailwind config ?

File: components/molecules/cards/card-project/adapters/default/default.variants.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { tv } from "tailwind-variants";
+
+export const CardProjectDefaultVariants = tv({
+  slots: {
+    base: "flex flex-row items-stretch justify-start gap-2 p-2",
+  },
+  variants: {},
+  defaultVariants: {},
```
Comment (haydencleary) [inline components/molecules/cards/card-project/adapters/default/default.variants.ts:8]: Remove ?

File: components/molecules/cards/card-project/card-project.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { CardProjectLoading } from "components/molecules/cards/card-project/card-project.loading";
+
+import { CardProjectPort } from "./card-project.types";
+import { CardProject } from "./variants/card-project-default";
+
+type Story = StoryObj<typeof CardProject>;
+
+const defaultPort: CardProjectPort<"div"> = {
```
Comment (haydencleary) [inline components/molecules/cards/card-project/card-project.stories.tsx:10]:
```suggestion
const defaultProps: CardProjectPort<"div"> = {
```

File: components/molecules/cards/card-project/card-project.types.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  paperProps?: Partial<PaperPort<C>>;
+  htmlProps?: ComponentPropsWithoutRef<C>;
+  classNames?: Partial<ClassNames>;
+  avatarProps: AvatarPort;
+  title?: ReactNode;
+  description?: ReactNode;
+  topTags?: Array<TagPort<"div">>;
+  bottomTags?: Array<TagPort<"div">>;
+  primaryAction?: ButtonPort<"button">;
+  secondaryAction?: ButtonPort<"button">;
```
Comment (haydencleary) [inline components/molecules/cards/card-project/card-project.types.ts:25]:
```suggestion
  titleProps?: ReactNode;
  description?: ReactNode;
  topTags?: Array<TagPort<"div">>;
  bottomTags?: Array<TagPort<"div">>;
  primaryActionProps?: ButtonPort<"button">;
  secondaryActionProps?: ButtonPort<"button">;
```
We have similar naming in other molecules, same issue in your other PR. We can discuss this.

File: components/molecules/cards/card-project/adapters/default/default.adapter.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  title,
+  description,
+  topTags,
+  paperProps = {},
+  bottomTags = [],
+  primaryAction,
+  secondaryAction,
+  ...props
+}: CardProjectPort<C>) {
+  const { ...htmlProps } = props;
```
Comment (haydencleary) [inline components/molecules/cards/card-project/adapters/default/default.adapter.tsx:27]: You could just extract `htmlProps` with the others above, no ?

File: components/molecules/cards/card-project/adapters/default/default.adapter.tsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+          <Typo size={"m"} weight="medium" as={"div"}>
+            {title}
+          </Typo>
+          <div className="flex w-full flex-1 justify-end gap-1">
+            {topTags?.map((t, key) => (
+              <Tag key={key} size={"xs"} shape={"round"} style={"outline"} color="blue" {...t} />
+            ))}
+          </div>
+        </div>
+        <div className="w-full">
```
Comment (haydencleary) [inline components/molecules/cards/card-project/adapters/default/default.adapter.tsx:55]: Should hide this div if there is no description/

File: components/molecules/cards/card-project/card-project.types.ts (hunk 5 above)
Comment (haydencleary) [reply components/molecules/cards/card-project/card-project.types.ts:25]: And shouldn't the title be required ?

File: components/atoms/typo/adapters/default/default.variants.ts (hunk 2 above)
Comment (alexbeno) [reply components/atoms/typo/adapters/default/default.variants.ts:9]: yes that i have to done in a first iteration but the font size defined in the tailwind config doesn't work..

**Review by haydencleary — APPROVED** [review]

**Pull request merged**